jenkins/unit-integration-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -v
+mkdir -p $HOME/.sbt/launchers/0.13.8/
+test -r $HOME/.sbt/launchers/0.13.8/sbt-launch.jar || curl -L -o $HOME/.sbt/launchers/0.13.8/sbt-launch.jar http://dl.bintray.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.8/sbt-launch.jar
+java -jar $HOME/.sbt/launchers/0.13.8/sbt-launch.jar -Dsbt.log.noformat=true "; clean; coverage; doc; assembly"
+java -Dfile.encoding=utf-8 -jar $HOME/.sbt/launchers/0.13.8/sbt-launch.jar  -Dsbt.log.noformat=true "; coverageReport; coveralls"
```

Comment:
If you merge the two commands (In quotations) together, sbt won't reload and a good amount of build time would be saved.